### ISSUE TITLE
[Accessibility] Details List drag and drop is not keyboard accessible.

### ISSUE THREAD

**Issue by chadsowald** [microsoft/fluentui#11837]
### Environment Information

- **Package version(s)**: 7.7.0
- **Browser and OS versions**: Chrome latest on Win 10

### Describe the issue:

The built in details list drag and drop does not appear to be keyboard accessible.  I can't find any mention of it in the documentation and the example ( https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/draganddrop ) doesn't seem to be accessible.

### Please provide a reproduction of the issue in a codepen:

Just try the example: https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/draganddrop

#### Actual behavior:

Cannot initiate drag and drop via keyboard.

#### Expected behavior:

Can start and end drag and drop via keyboard, and also abort drag and drop via keyboard.

### Documentation describing expected behavior

These various examples all seem to, by default, let the user press SPACE and then start using the arrow keys to move items around.  The user then must press SPACE again to finish the drag and drop.  They can press ESC to abort the drag and drop.

https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/about/accessibility.md
https://codesandbox.io/s/k260nyxq9v  (via https://medium.com/salesforce-ux/4-major-patterns-for-accessible-drag-and-drop-1d43f64ebf09)

I imagine that's what users needing keyboard support are used to.

**Comment by khmakoto** [comment]
Hi @chadsowald, thanks for filing this issue! I don't think we currently have plans to add this kind of behavior to the `DetailsList` component but it would definitely be a cool thing to add to it. We'll keep this on our backlog for now.

**Comment by chadsowald** [comment]
Thanks @khmakoto.  Appreciate your quick reply and thanks for considering this for later on.  We'll look for either another library to help with drag/drop or reconsider our UX for reordering.

**Comment by cschlechty** [comment]
@dzearing can we close as a backlog item? Not required feature at this time.

**Comment by xugao** [comment]
@cschlechty - I actually got help from a11y tester to assess impact of this one

This is a medium impact issue and here is the [MAS rule](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b8492c4eb-c179-40ae-8777-cd044ed725a2%7d&CID=cdffd8b6-a202-0907-c20d-c64bea52b143) we are violating here.

I think this is a bug which can surface again later during a11y testing and impact our scorecard. we should have some plan here on how to handle this

**Comment by cschlechty** [comment]
If there's an accessible way to achieve the outcome, that's sufficient.

**Comment by jimmy3912msncom** [comment]
Is there an ETA on this?

**Comment by richardtagger** [comment]
+1 @dzearing is there a way we can have this prioritized, we would like to use a Drag and Drop Details List but it must be usable by all users. @xugao makes good points here.

**Comment by dzearing** [comment]
Adding @JustSlone and @paulgildea for visibility.

**Comment by JustSlone** [comment]
There is not currently and ETA on the feature. We are not adding features to Details List. However, given this impacts accessibility we should look into it to see what we can do.

**Comment by Saikumarga** [comment]
We are using details list drag and drop functionality and we want it to be accessible to all users. Is there an ETA on this?
